core/src/test/java/com/linecorp/armeria/internal/annotation/AnnotatedHttpServiceBuilderTest.java

```diff
@@ -68,12 +67,13 @@ public void root() {}
             public void root() {}
         });
 
-        Server.builder().annotatedService(new Object() {
-            @Options
-            @Get
-            @Post("/")
-            public void root() {}
-        });
+        // TODO: Check if we need to keep supporting this
+        //        Server.builder().annotatedService(new Object() {
+        //            @Options
+        //            @Get
+        //            @Post("/")
+        //            public void root() {}
+        //        });
```

Comment:
I think it's fine to stop supporting it and just raise an exception. How about adding  the test cases for it?